Update fetch metadata article to state that Firefox supports FM (#6810)
GoogleChrome/web.dev: src/site/content/en/blog/fetch-metadata/index.md

## Browser compatibility {: #compatibility }

> Fetch Metadata request headers are supported as of Chrome 76 and in other Chromium-based browsers, and as of Firefox 90.
> See [Browser compatibility](https://developer.mozilla.org/docs/Web/HTTP/Headers/Sec-Fetch-Site#Browser_compatibility) for up-to-date browser support information.
> 

Reviewer: ```suggestion
Fetch Metadata request headers are supported as of Firefox 90 and as of Chrome 76 in all Chromium-based browsers.

{% BrowserCompat 'http.sec-fetch-site' %}

```

Reviewer: Note to self: look up the correct attribute for the compat widget here

## Browser compatibility {: #compatibility }

Fetch Metadata request headers are supported as of Firefox 90 and as of Chrome 76 in all Chromium-based browsers.

> {% BrowserCompat 'http.sec-fetch-site' %}

Reviewer 2: ```suggestion
{% BrowserCompat 'http.headers.sec-fetch-site' %}
```
